astropy/cosmology/flrw/base.py

```diff
@@ -44,6 +51,13 @@ def quad(*args, **kwargs):
 _kB_evK = const.k_B.to(u.eV / u.K)
 
 
+# typing
+_FLRWT = TypeVar("_FLRWT", bound="FLRW")
+_FFLRWT = TypeVar("_FFLRWT", bound="FlatFLRWMixin")
```

Comment:
Would it be a good idea to make the type variable name more similar to the corresponding class name?
```suggestion
_FlatFLRWMixinT = TypeVar("_FlatFLRWMixinT", bound="FlatFLRWMixin")
```